moveit_core/collision_detection/include/moveit/collision_detection/collision_detector_allocator.h

```diff
@@ -45,7 +45,9 @@ namespace collision_detection
 {
 MOVEIT_CLASS_FORWARD(CollisionDetectorAllocator);
 
-/** \brief An allocator for a compatible CollisionWorld/CollisionRobot pair. */
+/** \brief An allocator for a compatible CollisionWorld/CollisionRobot pair.
+ *
+ *  It ensures that a valid CollisionWorld / CollisionRobot combination is used in the planning scene. */
```

Comment:
This new comment is not very helpful. The allocator doesn't ensure anything, does it?